services/status/api.go

```diff
@@ -54,18 +54,27 @@ type SignupRequest struct {
 
 // SignupResponse : json response returned by status_signup.
 type SignupResponse struct {
-	Address  string `json:"address"`
-	Pubkey   string `json:"pubkey"`
-	Mnemonic string `json:"mnemonic"`
+	WalletAddress string `json:"address"`
```

Comment:
Compatibility breaks here...?

Reply:
it shouldn't break because the json field has still the same name, but I added now both address and walletAddress